src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoConnectedTest.java

```diff
@@ -148,15 +148,21 @@ public void testSynapseQueryCSV() throws Exception {
     IngestRequestModel ingestRequestModel =
         new IngestRequestModel().format(FormatEnum.CSV).csvSkipLeadingRows(2);
     String ingestFileLocation =
-        "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/azure-simple-dataset-ingest-request.csv";
+        synapseUtils.ingestRequestURL(
+            testConfig.getSourceStorageAccountName(),
+            testConfig.getIngestRequestContainer(),
+            "azure-simple-dataset-ingest-request.csv");
```

Comment:
Are these ingest files already in the test azure storage account?

Reply:
Yes - These are in the "connectedapp" managed application and the integration ones are in the "integrationapp" managed application.